dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceConfiguration.java

```diff
@@ -148,6 +148,10 @@ public class DockstoreWebserviceConfiguration extends Configuration {
 
     private String checkUrlLambdaUrl;
 
+    private String sourceFilePathRegex;
```

Comment:
Is there a reason why these don't have defaults like `dashboard`?
Think the default could just be your original suggestion

Reply:
I think I'm ok with just using your restrictions and then loosening later if we get the requests. I think you mentioned that we haven't actually run into any problematic paths yet.

I find it's usually easier in these cases to loosen than tighten.